frontend/model/state.js

```diff
@@ -250,6 +263,25 @@ const getters = {
     const currency = currencies[getters.groupSettings.mincomeCurrency]
     return currency && currency.symbolWithCode
   },
+  groupIncomeDistribution (state, getters) {
+    const profiles = getters.groupMembers
+    const mincomeAmount = getters.groupMincomeAmount
+    const currentIncomeDistribution = []
+    for (const username in profiles) {
+      const profile = profiles[username]
+      const incomeDetailsType = profile && profile.groupProfile.incomeDetailsType
```

Comment:
Could use !!profile.groupProfile.incomeDetailsType

Reply:
No since that would return a boolean, and I need its value to retrieve the income/pledgeAmount